<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>blocks2stacks</class>
 <widget class="QMainWindow" name="blocks2stacks">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>365</width>
    <height>307</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>blocks2stacks</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QGroupBox" name="blocksGroupBox">
        <property name="title">
         <string>Blocks</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Blocks</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="blocksSpinBox_2">
           <property name="minimum">
            <number>0</number>
           </property>
           <property name="maximum">
            <number>99999</number>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="stacksGroupBox">
        <property name="title">
         <string>Stacks</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <widget class="QLabel" name="stacksLabel">
             <property name="text">
              <string>Stacks</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="stacksSpinBox">
             <property name="maximum">
              <number>99999</number>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Fixed</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>5</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QLabel" name="blocksLabel">
             <property name="text">
              <string>Blocks</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="blocksSpinBox">
             <property name="maximum">
              <number>99999</number>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>365</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
